### PR TITLE
Implement credit cards payments (PayU) in Colombia

### LINKED ISSUE #439: Implement credit card for Colombia (PayU Integration)
EBANX is processing credit card in Colombia and we need implement it on plugin. 

We need to discuss more about the instalments, because this payment method has different instalment values.

More informations about this payment method:

- Country: Colombia
- Payment method: Credit Card (Tarjeta de Crédito)
- Instalments: Supported installments for Crédito Fácil Codensa: 6, 12, 18, 24, 36 and 48.
- DNI is required.

### PR DESCRIPTION
**Pull request by guilhermepiovesan**
Closes #439

### CODE REVIEW

**Comment by guilhermepiovesan** [comment]
Guys, we cannot test this new payment method because the sandbox still does not accept

**Comment by cezarlz** [comment]
@guilhermepiovesan It's missing files... can you review them?

**Comment by guilhermepiovesan** [comment]
What is the minimal instalment value for acquirers to approve based on COP?

**Comment by cezarlz** [comment]
@guilhermepiovesan For while, we will use only 12 instalments, same as another countries.

File: woocommerce-gateway-ebanx/.vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place your settings in this file to overwrite default and user settings.
+{
+}
```
Comment (SparK-Cruz) [inline woocommerce-gateway-ebanx/.vscode/settings.json]: No way!!!
Keep your editor files out of the repository 😠

File: woocommerce-gateway-ebanx/templates/ebanx-credit-card-co/payment-form.php [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            </div>
+    	<?php endif;?>
+    </section>
+</div>
+
+<script>
+	// Custom select fields
+	if ('jQuery' in window && 'select2' in jQuery.fn) {
+		jQuery('select.ebanx-select-field').select2();
+	}
```
Comment (SparK-Cruz) [inline woocommerce-gateway-ebanx/templates/ebanx-credit-card-co/payment-form.php]: We don't use those anymore. Try merging develop and check `wc-enhanced-select` css class

File: woocommerce-gateway-ebanx/.vscode/settings.json (hunk 1 above)
Comment (gpressutto5) [inline woocommerce-gateway-ebanx/.vscode/settings.json]: 🤣 🤣 🤣  Roger has gotten pistol

**Pull request closed without merging**